From a2cc0060dd0ff131315fa44f9259afb921656137 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Fri, 24 Dec 2021 12:23:43 +0100
Subject: [PATCH 6/8] user32: Remove FIXME from RegisterDeviceNotificationA.

There's no conversion to do, the device name in the filter is ignored.
---
 dlls/user32/input.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/dlls/user32/input.c b/dlls/user32/input.c
index 67ca7e178f2..fde4b9965ed 100644
--- a/dlls/user32/input.c
+++ b/dlls/user32/input.c
@@ -1186,13 +1186,9 @@ extern BOOL WINAPI I_ScUnregisterDeviceNotification( HDEVNOTIFY handle );
  *
  * See RegisterDeviceNotificationW.
  */
-HDEVNOTIFY WINAPI RegisterDeviceNotificationA(HANDLE hRecipient, LPVOID pNotificationFilter, DWORD dwFlags)
+HDEVNOTIFY WINAPI RegisterDeviceNotificationA( HANDLE handle, void *filter, DWORD flags )
 {
-    TRACE("(hwnd=%p, filter=%p,flags=0x%08x)\n",
-        hRecipient,pNotificationFilter,dwFlags);
-    if (pNotificationFilter)
-        FIXME("The notification filter will requires an A->W when filter support is implemented\n");
-    return RegisterDeviceNotificationW(hRecipient, pNotificationFilter, dwFlags);
+    return RegisterDeviceNotificationW( handle, filter, flags );
 }
 
 /***********************************************************************
-- 
2.34.1

